### DISCUSSION TITLE
The random lyrics at the bottom of the desktop

### DISCUSSION THREAD

**Discussion by Bill-Klay** [hyprwm/Hyprland#3482] [Q&A] [answered]
Vanilla Hyprland comes with a desktop with the hyprland logo and randomly appearing hyprchan with a new song lyric at the bottom of the desktop wallpaper every time the machine is booted. After installing swww on my Arch install I've been trying to get that lyrics thing working again, but to no avail have I found the method Hyprland uses. I've configured conky in the past and variety, but any list that Hyprland takes its pick from? Does Hyprland use Hyprpaper as the default desktop service? Any guidance in the direction will be appreciated.
Thank you.

**Comment by pavanbhat1999** [comment, marked as answer]
`hyprctl splash`

you have to configure either waybar or eww to use this value over your bg image.

**Comment by MightyPlaza** [comment]
hyprpaper allows splash to be displayed on top of the wallpaper

**Comment by pavanbhat1999** [comment]
oh yes actually.. My bad I haven't checked properly hyprpaper.

`splash=true`
in hyprpaperconfig and good to go.

**Comment by Bill-Klay** [comment]
Thanks a bunch! I'll get it done with waybar since I need swww and its dynamic bg animation.
